### DISCUSSION TITLE
Docker容器的端口是否可自定义

### DISCUSSION THREAD

**Discussion by ouooo** [lobehub/lobehub#1954] [Self Hosting] [answered]
大佬好，这是一个不那么重要的问题，只是我在部署时想要**自定义容器端口号和宿主机端口号一致**，但是我不确定是否可以这样做，如果可以自定，还需要再环境变量中添加PORT="端口号"然后再进行端口映射吗？
还有一个问题是部署的指令里**没有映射存储卷**，是不是数据不需要保存呀

**Comment by cy948** [comment, marked as answer]
1. 端口映射可以自行指定，只需更改 docker 容器启动指令。
2. 容器没有数据需要保存。
